packages/create-cloudflare/e2e-tests/helpers.ts

```diff
@@ -79,6 +79,7 @@ export type RunnerConfig = {
 		route: string;
 		expectedText: string;
 	};
+	verifyTest?: boolean;
```

Comment:
might be useful to describe what these `verify<X>`s do via a comment for each?

Reply:
Added description on https://github.com/cloudflare/workers-sdk/pull/7827/commits/26d37a8b49762a209ac49e22496399d095b63f94